spec/concepts/tribunal_commerce/stock/operation/import_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec::Matchers.define_negated_matcher :not_change, :change
```

Comment:
j'aurai bien vu ça dans `spec/support/matchers/negated_matchers.rb` (ou un autre nom), mais pas directement dans les specs

Reply:
Pour le moment j'en ai qu'un et je l'utilise qu'une fois : dans ces specs précisément (juste pour pouvoir combiner mon *expectation* avec un *and*). Le mettre ailleurs ne fait rien gagner atm, mais dans l'idée je suis d'accord.